recipe/meta.yaml

```diff
@@ -52,25 +54,35 @@ requirements:
     - cuda-compat      {{ cuda_compiler_version }}        # [(gpu_variant or "").startswith('cuda') and (gpu_variant != "cuda-11") and linux]
     - openblas-devel {{ openblas }}                       # [(not (gpu_variant or "").startswith('cuda')) and blas_impl == "openblas"]
     - mkl-devel {{ mkl }}                                 # [(not (gpu_variant or "").startswith('cuda')) and blas_impl == "mkl"]
+    - intel-openmp {{ mkl }}                              # [(not (gpu_variant or "").startswith('cuda')) and blas_impl == "mkl"]
```

Comment:
how come the intel openmp is needed now? The cmake config has only changed for osx

Reply:
OpenMP has been enabled by default upstream.
https://github.com/ggerganov/llama.cpp/blob/b3184/CMakeLists.txt#L134